### ISSUE TITLE
Use the database driver conditionally on specific collections

### ISSUE THREAD

**Issue by nfragkos** [statamic/eloquent-driver#11]
Is it possible to have a custom entry class and use it only for a specific collection? I love the idea to have my pages on the filesystem but for a blog collection with thousands of entries to use the database.

**Comment by edalzell** [comment]
Not possible yet, because all collections share the same repository, but I am trying to figure out how it all works so I can start down that path.

**Comment by nfragkos** [comment]
Hi I really needed that functionality so I created an addon called [Eloquenty](https://github.com/nfragkos/statamic-eloquenty). I currently marked it as pre-release as I'm looking for testers and feedback.

**Comment by ryanmitchell** [comment]
Is this not what runway does? https://statamic.com/addons/double-three-digital/runway

**Comment by edalzell** [comment]
> Is this not what runway does? [statamic.com/addons/double-three-digital/runway](https://statamic.com/addons/double-three-digital/runway)

No, runway brings Laravel models into antlers and the CP, the other packages uses the db to store Statamic Entries.

**Comment by ryanmitchell** [comment]
Yeah I guess it was more a question of why :)

If you want entries in the DB use this, if you want them "split" but the same format set up a database view, if you want different models use runway. Each to their own I guess :)
